docs/source/certs_management.md

```diff
@@ -379,3 +379,77 @@ Certificate:
          9c:02:20:51:33:42:5e:a0:8a:2a:ec:f5:83:46:f0:99:6a:7e:
          eb:a8:97:1f:30:99:9d:ae:8d:ef:36:07:da:bb:67:ed:80
 ```
+
+## Certificate Renewal
+
+Enrollment certificates and TLS certificates for all nodes and clients should be updated before they expire in order to avoid disruption of service.
+Obtain updated certificates from the issuing Fabric Certificate Authority (CA) by using the Fabric CA enroll or reenroll function.
+The reenroll function allows the reuse of an existing private key, which is especially important for orderer TLS certificates.
+
+### Certificate Renewal of Expired Certificates
+
+If certificates expire before reenrollment you must enable the `reenrollignorecertexpiry` option on the Fabric CA server prior to reenrollment.
+
+Upgrade all Fabric CA nodes to v1.5.5 or above. Then add the `reenrollignorecertexpiry` parameter to the fabric-ca-server-config.yaml, as follows:
+
+```
+ca:
+  certfile: /crypto/tlsca/cert.pem
+  chainfile: /crypto/tlsca/chain.pem
+  keyfile: /crypto/tlsca/key.pem
+  name: tlsca
+  reenrollignorecertexpiry: true
+```
+
+Alternatively, `reenrollignorecertexpiry` can be also be set with an environment variable or Fabric CA server startup flag:
+
+* Set the FABRIC_CA_SERVER_CA_REENROLLIGNORECERTEXPIRY environment variable to true at CA startup
+* Start the CA with the flag --ca.reenrollignorecertexpiry=true
+
+Restart the Fabric CA servers.
+
+### Renew Peer Certificates
+
+For peer enrollment certificates and TLS certificates you can use either the `enroll` or `reenroll` function from the Fabric CA client. Use `reenroll` if you would like to reuse a private key by passing the `csr.keyrequest.reusekey` option and indicating the `mspdir` location of the existing private key (the private key is found in the <mspdir>/keystore directory):
+
+```
+fabric-ca-client enroll --<OTHER-OPTIONS>
+
+fabric-ca-client reenroll --csr.keyrequest.reusekey --mspdir <LOCATION-OF-IDENTITY'S-MSP-DIRECTORY-THAT-CONTAINS-THE-EXISTING-PRIVATE-KEY> --<OTHER-OPTIONS>
+```
+
+Replace the peer's enrollment certificate or TLS certificate.
+
+The peer's enrollment certificate is configured to be in the peer's `<peer.mspConfigPath>/signcerts` directory (if private key was not reused the new private key should go to `<peer.mspConfigPath>/keystore` directory).
+
+The peer's TLS certificate is configured to be in the peer's `<peer.tls.cert.file>` directory/file (if private key was not reused the new private key should go to `<peer.tls.key.file>`).
+
+Restart the peer.
+
+### Renew orderer certificates
```

Comment:
initial caps for consistency w others. Renew Orderer Certificates. (that said, the trend is towards sentence style headings)